cli/src/cast.rs

```diff
@@ -634,7 +634,7 @@ async fn main() -> eyre::Result<()> {
                     .expect("failed to generate vanity wallet");
 
                 println!(
-                    "Successfully created new keypair in {} seconds.\nAddress: {}.\nPrivate Key: {}.",
+                    "Successfully created new keypair in {} seconds.\nAddress: {}\nPrivate Key: {}",
```

Comment:
I noticed `cast wallet new` is not the only command that outputs Address and Private Key. `cast wallet vanity` also outputs these fields. I removed trailing periods here too for consistency.